app/src/Components/AddScore.js

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+
+const AddScore = ({
```

Comment:
I'm not sure how universal it is, but I would usually name components after with purely nouns, without verbs, so something like: `Scoreboard` or `ScoreTracker` rather than `AddScore` but the <Verb><Noun> convention you are using might be a functional component thing? Did anyone encourage you to name components this way?